pkgs/tools/X11/sct/default.nix

```diff
@@ -4,7 +4,7 @@ stdenv.mkDerivation rec {
   buildInputs = [libX11 libXrandr];
   src = fetchurl {
     url = http://www.tedunangst.com/flak/files/sct.c;
-    sha256 = "1bivy0sl5v1jsq4jbq6p9hplz6cvw4nx9rc96p2kxsg506rqllc5";
+    sha256 = "1321ajd1ph0hyj6pi76ylfcb1csj100lqflrb7n44ni5ybxmy3j2";
```

Comment:
Discussion regarding the checksum and the source code can be found in issue #17163 
The code seems unmaintained, yet an unknown (probably small change) in the code caused failed builds as the checksum had changed. 

The checksum is updated for now, however, this is unpractical and potentially unsafe so any future changes might warrant a fork of the (feature complete) project. The code is under public domain.